src/CAREUI/display/Count.tsx

```diff
@@ -18,7 +19,7 @@ export default function CountBlock(props: Props) {
         </div>
         <div>
           <dt className="mb-1 truncate text-sm font-semibold text-secondary-700">
-            {props.text}
+            {t(props.text)}
```

Comment:
wrap with `t` where the key is being used; not a variable.

```suggestion
            {props.text}
```